src/worker/jobs/dump_db.rs

```diff
@@ -202,15 +221,22 @@ pub fn run_psql(script: &Path, database_url: &str) -> anyhow::Result<()> {
     Ok(())
 }
 
-fn create_tarball(export_dir: &Path, prefix: &Path) -> anyhow::Result<tempfile::NamedTempFile> {
-    debug!("Creating tarball file");
-    let tempfile = tempfile::NamedTempFile::new()?;
-    let encoder = flate2::write::GzEncoder::new(tempfile.as_file(), flate2::Compression::default());
+fn create_archives(
+    export_dir: &Path,
+    tarball_prefix: &Path,
+) -> anyhow::Result<(tempfile::NamedTempFile, tempfile::NamedTempFile)> {
```

Comment:
Would it be better to return a struct with explicit named fields? Without seeing the implementation, it's difficult to determine the order of zip or tarball from the current return type.

Reply:
done :)